app/models/like.rb

```diff
@@ -1,4 +1,4 @@
 class Like < ApplicationRecord
-  belongs_to :users
-  belongs_to :posts
+  belongs_to :user
+  belongs_to :post
```

Comment:
- [ ] It will be nice to include a method that updates the likes counter for a post. A method similar to this:

```ruby
# app/models/like.rb
...

def update_likes_counter
  post.increment!(:likes_counter)
end
```
I can see you implemented them in the post.rb. So you need to refactor your code and place the methods in their respective models.